test/e2e/framework/framework.go

```diff
@@ -266,6 +266,13 @@ func (f *Framework) AfterEach() {
 		LogPodsWithLabels(f.Client, api.NamespaceSystem, ImagePullerLabels)
```

Comment:
@dchen1107 Good point. Actually, I was thinking we should move the whole print-prepull-container-logs from being printed AfterEach failure to be printed only once, just like the nethealth container.
Not sure if we need the output of the prepull containers after every failed test.